docs/guides/Guide-Basic-Example.md

```diff
@@ -0,0 +1,235 @@
+
+```js
+import React from 'react';
+import {
+  AppRegistry,
+  Text,
+} from 'react-native';
+import { StackNavigator } from 'react-navigation';
+
+class HomeScreen extends React.Component {
+  static navigationOptions = {
+    title: 'Welcome',
+  };
+  render() {
+    return <Text>Hello, Navigation!</Text>;
+  }
+}
+
+export const SimpleApp = StackNavigator({
+  Home: { screen: HomeScreen },
+});
+
+AppRegistry.registerComponent('SimpleApp', () => SimpleApp);
+```
+
+If you used `create-react-native-app` the already existing `App.js` will be modified to
+
+```js
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { StackNavigator } from 'react-navigation';
+
+class HomeScreen extends React.Component {
+  static navigationOptions = {
+    title: 'Welcome'
+  };
+  render() {
+    return <Text>Hello, Navigation!</Text>;
+  }
+}
+
+const SimpleApp = StackNavigator({
+  Home: { screen: HomeScreen }
+});
+
+export default class App extends React.Component {
+  render() {
+    return <SimpleApp />;
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center'
+  }
+});
+
+```
+
+The `title` of the screen is configurable on the [static `navigationOptions`](/docs/navigators/navigation-options), where many options can be set to configure the presentation of the screen in the navigator.
+
+Now the same screen should appear on both iPhone and Android apps:
+
+```phone-example
+first-screen
+```
+
+## Adding a New Screen
+
+In our `App.js` file, let's add a new screen called `ChatScreen`, defining it under `HomeScreen`:
+
+```js
+// ...
+
+class HomeScreen extends React.Component {
+    //...
+}
+
+class ChatScreen extends React.Component {
+  static navigationOptions = {
+    title: 'Chat with Lucy',
+  };
+  render() {
+    return (
+      <View>
+        <Text>Chat with Lucy</Text>
+      </View>
+    );
+  }
+}
+
+```
+
+We can then add a button to our `HomeScreen` component that links to `ChatScreen`: we need to use the provided method `navigate` (from the [screen navigation prop](/docs/navigators/navigation-prop)) by giving it the `routeName` of the screen we want to reach, in this case `Chat`.
+
+```js
+class HomeScreen extends React.Component {
+  static navigationOptions = {
+    title: 'Welcome',
+  };
+  render() {
+    const { navigate } = this.props.navigation;
+    return (
+      <View>
+        <Text>Hello, Chat App!</Text>
+        <Button
+          onPress={() => navigate('Chat')}
+          title="Chat with Lucy"
+        />
+      </View>
+    );
+  }
+}
+```
+
+(*don't forget to import View and Button from react-native: * `import { AppRegistry, Text, View, Button } from 'react-native';`)
```

Comment:
I think the second \*, before the import, needs to be attached to : or it won't be italic (and this is probably a mistake I did when I updated this file) XD